modules/installation-nutanix-ccm.adoc

```diff
@@ -0,0 +1,85 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_nutanix/installing-nutanix-installer-provisioned.adoc
+
+:_content-type: PROCEDURE
+[id="nutanix-ccm-config_{context}"]
+= Adding the configmaps and secret required for Nutanix CCM
```

Comment:
"config map" should be two words when used as a [general reference](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#general-references).